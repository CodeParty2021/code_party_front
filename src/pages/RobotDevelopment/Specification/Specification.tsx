import React from "react";
import { Link } from "react-router-dom";
type Prop = {};

export const Specification: React.FC<Prop> = () => {
  return (
    <div>
      <h1>ロボットの設計仕様書</h1>
      <Link to="/robot-development/top">
        ロボット開発モードのトップ画面に戻る
      </Link>
    </div>
  );
};
