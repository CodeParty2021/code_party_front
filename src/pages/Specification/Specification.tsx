import React from "react";

type Prop = {};

export const Specification: React.FC<Prop> = () => {
  return (
    <div>
      <h1>ロボットの設計仕様書</h1>
    </div>
  );
};