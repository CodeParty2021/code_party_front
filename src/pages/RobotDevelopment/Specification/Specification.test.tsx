import React from "react";
import { shallow } from "enzyme";
import { Specification } from "./Specification";
describe("<Specification />", () => {
  it("snapshot test", () => {
    const wrapper = shallow(<Specification />);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
