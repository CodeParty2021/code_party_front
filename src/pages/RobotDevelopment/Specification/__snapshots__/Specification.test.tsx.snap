// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Specification /> snapshot test 1`] = `
Array [
  <div>
    <h1>
      ロボットの設計仕様書
    </h1>
    <Link
      to="/robot-development/top"
    >
      ロボット開発モードのトップ画面に戻る
    </Link>
  </div>,
]
`;
